firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/ApkInstallerTests.java

```diff
@@ -65,9 +65,9 @@ public void installApk_currentActivityNotNull_InstallIntentOnCurrentActivity() {
 
   @Test
   public void installApk_currentActivityNull_InstallNotPrompted() {
-    when(mockLifecycleNotifier.getCurrentActivity()).thenReturn(null);
+    when(mockLifecycleNotifier.getForegroundActivity()).thenReturn(Tasks.forResult(null));
```

Comment:
The foreground activity should never be null. We should delete this test case.

Reply:
talked offline will delete